### PR TITLE
Added a setting for log output when destroy is executed #452

### PR DESCRIPTION
**Pull request by btootomoyuk**
Please review #452.

Confirmation

Confirmed that the addition of the setting causes the output of the log on destroy, which was not output before the modification.

### CODE REVIEW

File: src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -4,6 +4,15 @@
...
     version="3.0">
 
+    <context-param>
+        <param-name>logbackDisableServletContainerInitializer</param-name>
+        <param-value>true</param-value>
+    </context-param>
+
+    <listener>
+        <listener-class>ch.qos.logback.classic.servlet.LogbackServletContextListener</listener-class>
+    </listener>
```
Comment (btkatoufm) [inline src/main/webapp/WEB-INF/web.xml:14]: Do you write in the order of `listener`,` context-param`? 
Do you write in the order of `context-param`,` listener`?
It looks like it's not unified.

Comment (btootomoyuk) [reply src/main/webapp/WEB-INF/web.xml]: db18cc216024ac79cc5eb74c07ad6f9b4d00b5e8
Sorted by listener, context-param, according to the original order.

Comment (btkatoufm) [reply src/main/webapp/WEB-INF/web.xml]: - [ ] The behavior may change by changing the registration order of listeners. Is this taken into consideration?
At least to resolve this ISSUE, you must first register `LogbackServletContextListener`.

**Review by btkatoufm — APPROVED** [review]

File: src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -4,9 +4,23 @@
...
     </listener>
+
+    <listener>
+        <listener-class>org.terasoluna.gfw.web.logging.HttpSessionEventLoggingListener</listener-class>
+    </listener>
+
+    <context-param>
+        <param-name>logbackDisableServletContainerInitializer</param-name>
+        <param-value>true</param-value>
+    </context-param>
```
Comment (yoshikawaa) [inline src/main/webapp/WEB-INF/web.xml:22]: - [ ] It's very bad. This fix obscures the effect of `contextConfigLocation`, so I recommend modifying it as follows:

```
context-param:logbackDisableServletContainerInitializer (add)
listener:LogbackServletContextListener (add)
listener:ContextLoaderListener + context-param:contextConfigLocation (no edit)
listener:HttpSessionEventLoggingListener (no edit)
```

This order is based on the following policy:

* `context-param` affects the entire application.
* Define `context-param` at the top that is not related to a particular listener.
* Define `context-param` associated with a particular listener immediately after the listener to clarify the relationship.

**Review by yoshikawaa — COMMENTED** [review]
I recommend re-fixing this again.

File: src/main/webapp/WEB-INF/web.xml (hunk 2 above)
Comment (btkatoufm) [reply src/main/webapp/WEB-INF/web.xml:22]: Fixed it with https://github.com/terasolunaorg/terasoluna-gfw-web-blank/pull/478 .

**Pull request merged**